File: app/src/main/java/com/pavelprymak/propodcast/presentation/adapters/PodcastClickListener.java
package com.pavelprymak.propodcast.presentation.adapters;

import android.view.View;

import com.pavelprymak.propodcast.model.network.pojo.podcasts.PodcastItem;
import com.pavelprymak.propodcast.model.network.pojo.search.ResultsItem;

public interface PodcastClickListener {
    void onPodcastItemClick(String podcastId);

    void onPodcastMoreOptionsClick(PodcastItem podcastItem, View v);

}

File: app/src/main/java/com/pavelprymak/propodcast/presentation/adapters/PodcastClickListener.kt
package com.pavelprymak.propodcast.presentation.adapters

import android.view.View

import com.pavelprymak.propodcast.model.network.pojo.podcasts.PodcastItem

interface PodcastClickListener {
    fun onPodcastItemClick(podcastId: String)

    fun onPodcastMoreOptionsClick(podcastItem: PodcastItem, v: View)

}
